.github/workflows/config.yml

```diff
@@ -27,11 +27,15 @@ jobs:
       - name: Prepare Environment
         env:
           REF: ${{ github.ref }}
+          DEMISTO_BASE_URL: ${{ secrets.DEMISTO_BASE_URL }}
+          DEMISTO_API_KEY: ${{ secrets.DEMISTO_API_KEY }}
         run: |
           echo "PATH=/home/runner/.local/bin:$PATH" >> $GITHUB_ENV
+          echo "DEMISTO_BASE_URL="$DEMISTO_BASE_URL >> $GITHUB_ENV
+          echo "DEMISTO_API_KEY="$DEMISTO_API_KEY >> $GITHUB_ENV
           echo "PYTHONPATH=${GITHUB_WORKSPACE}/content:${GITHUB_WORKSPACE}:${PYTHONPATH}" >> $GITHUB_ENV
-          echo "ARTIFACTS_FOLDER=/home/runner/work/repository/repository/content/artifacts" >> $GITHUB_ENV
```

Comment:
Absolute paths like "/home/runner/work/repository/repository" are wrong. This doesn't take the repository name into consideration. Paths are now referenced by ${GITHUB_WORKSPACE} (e.g  ${GITHUB_WORKSPACE}/repository )